evennia/commands/cmdsethandler.py

```diff
@@ -351,6 +351,14 @@ def update(self, init_mode=False):
                     elif path:
                         cmdset = self._import_cmdset(path)
                         if cmdset:
+                            if cmdset.key == '_CMDSET_ERROR':
+                                # If a cmdset fails to load, check if we have a fallback path to use
+                                fallback_path = settings.CMDSET_FALLBACKS.get(path, None)
```

Comment:
Maybe there should be a warning message sent to players above a certain class (maybe Builders?) each time they try to use a fallback command.